src/auth/trusted/AuthSspi.cpp

```diff
@@ -363,7 +394,8 @@ int WinSspiServer::authenticate(Firebird::CheckStatusWrapper* status,
 		{
 			bool wheel = false;
 			string login;
-			sspi.getLogin(login, wheel);
+			Firebird::ObjectsArray<Firebird::string> grNames;
+			sspi.getLogin(login, wheel, grNames);
 			ISC_systemToUtf8(login);
```

Comment:
I suppose grNames use system encoding (something like cp1251 in case of Russia)? If yes they should be converted to utf8 like it's done for login.

Reply:
Ok i have fixed this - thank you